.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# Contributing to the CAF provider
+
+👍🎉 First off, thanks for taking the time to contribute! 🎉👍
+
+## Code of conduct
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
+For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
+contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+## What should I know before I get started?
+
+The current goal of the CAFprovider is to support the CAF landing zones but can also be used to standardize the naming convention of your projects. It is important to keep in mind that many of the design decisions on the provider have been made to accommodate the needs of CAF.
+
+To contribute to this project you are required to have at least go 1.13 installed in your system
+
+## Adding a new resource
+
+The process to add a new resource start with checking if the resource has already been implemented.
```

Comment:
"Please, find below the steps that should be followed to contribute:" or sth along those lines